include/aspect/heating_model/compositional_heating.h

```diff
@@ -99,7 +80,7 @@ namespace aspect
          * contribute to material properties like viscosity, heat production,
          * etc.
          */
-        std::vector<int> field_used_in_heat_production_averaging;
+        std::vector<bool> fields_used_in_heat_production_averaging;
```

Comment:
Is it correct that the length of this array is now equal to the number of compositional fields? What was it before -- the number of fields that are used, with each vector entry an index between zero and the number of compositional fields?